config/application.rb

```diff
@@ -29,5 +29,13 @@ class Application < Rails::Application
 
     # Automatically generate the `translation.js` files
     config.middleware.use I18n::JS::Middleware
+
+    # Use Rack::Cors to allow cross-origin requests in OpenAPI Swagger UI.
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*', headers: :any, methods: :any
```

Comment:
Will not this be applied to all routes? Rails provide [route constraints](https://guides.rubyonrails.org/routing.html#advanced-constraints) to limit what applies to specific routes (see https://tadhao.medium.com/constraints-in-rails-router-58f389031463)

Reply:
Thanks for this comment! #TIL

After rechecking, I think I will remove this, as this was set up for localhost. Currently, we can call our production URL :D

Reply:
Removed this config in https://github.com/nimblehq/nimble-survey-web/pull/89/commits/6391ef47b371a40433b500dd5c0775c31a735eca